### ISSUE TITLE
Release 4.4.0 - Docker Listener Module testing

### ISSUE THREAD

**Issue by nico-stefani** [wazuh/wazuh#14415]
# Docker Listener module testing

## Modules tests information
|                                  |                                             |
|----------------------------------|---------------------------------------------|
| **Main module tests issue**      | #14374  |
| **Version**                      | 4.4.0                                       |
| **Release candidate #**          |                                          |
| **Tag**                          |  |
| **Previous modules tests issue** |  |

## Test report procedure

All test results must have one the following statuses: 
|                                  |                                                        |
|-----------------|-------------------------------------------------------------------------|
| :green_circle:  | The test passed successfully. 										    |
| :yellow_circle: | The test passed but some enhancements need to be applied to the module. |
| :red_circle:    | The test failed.                                                        |
| :white_circle:  | The test does not apply to the given bucket type or service.            |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause. It must be included in the [Release 4.4.0 project](https://github.com/orgs/wazuh/projects/14/) for triage.

The data available in our Wazuh-dev environment for each of these supported services will be used. Additional logs may be created if they are necessary or there is insufficient data for testing.

### Conclusions

| Test  | Status Python 3.9 | Issue ref. |
|---|---|---|
| Testing docker-listener        |  :green_circle:   |  -  |

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @davidjiglesias
- [x] @CarlosRS9 
- [x] @nico-stefani

**Comment by nico-stefani** [comment]
## Tests

Manual testing was performed to ensure everything works after the change. The tests were carried out using both a Wazuh manager and a Wazuh agent both using the following configuration:
```
<wodle name="docker-listener">
    <interval>30s</interval>
    <attempts>5</attempts>
    <run_on_start>yes</run_on_start>
    <disabled>no</disabled>
</wodle>
```

The [logall_json option](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#logall-json) must be enabled in the `ossec.conf` file before starting the tests.
### Test cases Python 3.9


#### Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>

    2022/07/25 10:30:56 wazuh-modulesd:docker-listener[3740] wm_docker.c:45 at wm_docker_main(): INFO: Module docker-listener started.
    2022/07/25 10:30:56 wazuh-modulesd:docker-listener[3740] wm_docker.c:58 at wm_docker_main(): INFO: Starting to listening Docker events.
    2022/07/25 10:30:56 wazuh-modulesd:docker-listener[3740] wm_docker.c:62 at wm_docker_main(): DEBUG: Launching command 'wodles/docker/DockerListener'

</details>


#### Test docker commands

**Run a container with no cache available**
```
docker run ubuntu
```

<details><summary>Archives.json output</summary>

    {"timestamp":"2022-07-25T13:40:39.079+0000","rule":{"level":3,"description":"Docker: Container xenodochial_merkle created","id":"87901","firedtimes":4,"mail":false,"groups":["docker"],"pci_dss":["10.2.7"],"hipaa":["164.312.b"],"nist_800_53":["AU.14"],"tsc":["CC6.8","CC7.2","CC7.3"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.586869","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"create\", \"id\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"create\", \"Actor\": {\"ID\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"xenodochial_merkle\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439079319374}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"create","id":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","from":"ubuntu","Type":"container","Action":"create","Actor":{"ID":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","Attributes":{"image":"ubuntu","name":"xenodochial_merkle"}},"scope":"local","time":"1658756439","timeNano":"1658756439079319296.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:40:39.080+0000","rule":{"level":3,"description":"Docker: Attached local standard input, output, and error streams to container xenodochial_merkle","id":"87922","firedtimes":4,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.587959","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"attach\", \"id\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"attach\", \"Actor\": {\"ID\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"xenodochial_merkle\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439080189739}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"attach","id":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","from":"ubuntu","Type":"container","Action":"attach","Actor":{"ID":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","Attributes":{"image":"ubuntu","name":"xenodochial_merkle"}},"scope":"local","time":"1658756439","timeNano":"1658756439080189696.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:40:39.090+0000","rule":{"level":3,"description":"Docker: Network bridge connected","id":"87928","firedtimes":4,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.589022","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"connect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439089684121}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"connect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","name":"bridge","type":"bridge"}},"scope":"local","time":"1658756439","timeNano":"1658756439089684224.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:40:39.398+0000","rule":{"level":3,"description":"Docker: Container xenodochial_merkle started","id":"87903","firedtimes":4,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.589022","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"start\", \"id\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"start\", \"Actor\": {\"ID\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"xenodochial_merkle\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439397774539}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"start","id":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","from":"ubuntu","Type":"container","Action":"start","Actor":{"ID":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","Attributes":{"image":"ubuntu","name":"xenodochial_merkle"}},"scope":"local","time":"1658756439","timeNano":"1658756439397774592.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:40:39.456+0000","rule":{"level":7,"description":"Docker: Container xenodochial_merkle received the action: die","id":"87924","firedtimes":4,"mail":false,"groups":["docker"],"gdpr":["IV_32.2"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.590951","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"die\", \"id\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"die\", \"Actor\": {\"ID\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"Attributes\": {\"exitCode\": \"0\", \"image\": \"ubuntu\", \"name\": \"xenodochial_merkle\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439455675507}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"die","id":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","from":"ubuntu","Type":"container","Action":"die","Actor":{"ID":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","Attributes":{"exitCode":"0","image":"ubuntu","name":"xenodochial_merkle"}},"scope":"local","time":"1658756439","timeNano":"1658756439455675392.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:40:39.640+0000","rule":{"level":4,"description":"Docker: Network bridge disconnected","id":"87929","firedtimes":4,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756439.592033","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"disconnect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658756439, \"timeNano\": 1658756439640032031}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"disconnect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"d768626209b2e54b373fe25f4b41143549d5e334ac743ccff0eefc5b2adea86e","name":"bridge","type":"bridge"}},"scope":"local","time":"1658756439","timeNano":"1658756439640032000.000000"}},"location":"Wazuh-Docker"}

</details>

**Run the same image, this time having the image in cache**

<details><summary>Archives.json output</summary>

    {"timestamp":"2022-07-25T13:42:19.686+0000","rule":{"level":3,"description":"Docker: Container admiring_shirley created","id":"87901","firedtimes":5,"mail":false,"groups":["docker"],"pci_dss":["10.2.7"],"hipaa":["164.312.b"],"nist_800_53":["AU.14"],"tsc":["CC6.8","CC7.2","CC7.3"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756539.592964","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"create\", \"id\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"create\", \"Actor\": {\"ID\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"admiring_shirley\"}}, \"scope\": \"local\", \"time\": 1658756539, \"timeNano\": 1658756539686317331}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"create","id":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","from":"ubuntu","Type":"container","Action":"create","Actor":{"ID":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","Attributes":{"image":"ubuntu","name":"admiring_shirley"}},"scope":"local","time":"1658756539","timeNano":"1658756539686317312.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:42:19.687+0000","rule":{"level":3,"description":"Docker: Attached local standard input, output, and error streams to container admiring_shirley","id":"87922","firedtimes":5,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756539.596601","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"attach\", \"id\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"attach\", \"Actor\": {\"ID\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"admiring_shirley\"}}, \"scope\": \"local\", \"time\": 1658756539, \"timeNano\": 1658756539687179218}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"attach","id":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","from":"ubuntu","Type":"container","Action":"attach","Actor":{"ID":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","Attributes":{"image":"ubuntu","name":"admiring_shirley"}},"scope":"local","time":"1658756539","timeNano":"1658756539687179264.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:42:19.696+0000","rule":{"level":3,"description":"Docker: Network bridge connected","id":"87928","firedtimes":5,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756539.597658","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"connect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658756539, \"timeNano\": 1658756539696056244}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"connect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","name":"bridge","type":"bridge"}},"scope":"local","time":"1658756539","timeNano":"1658756539696056320.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:42:19.976+0000","rule":{"level":3,"description":"Docker: Container admiring_shirley started","id":"87903","firedtimes":5,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756539.598580","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"start\", \"id\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"start\", \"Actor\": {\"ID\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"admiring_shirley\"}}, \"scope\": \"local\", \"time\": 1658756539, \"timeNano\": 1658756539976196350}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"start","id":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","from":"ubuntu","Type":"container","Action":"start","Actor":{"ID":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","Attributes":{"image":"ubuntu","name":"admiring_shirley"}},"scope":"local","time":"1658756539","timeNano":"1658756539976196352.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:42:20.032+0000","rule":{"level":7,"description":"Docker: Container admiring_shirley received the action: die","id":"87924","firedtimes":5,"mail":false,"groups":["docker"],"gdpr":["IV_32.2"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756540.599581","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"die\", \"id\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"die\", \"Actor\": {\"ID\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"Attributes\": {\"exitCode\": \"0\", \"image\": \"ubuntu\", \"name\": \"admiring_shirley\"}}, \"scope\": \"local\", \"time\": 1658756540, \"timeNano\": 1658756540031720772}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"die","id":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","from":"ubuntu","Type":"container","Action":"die","Actor":{"ID":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","Attributes":{"exitCode":"0","image":"ubuntu","name":"admiring_shirley"}},"scope":"local","time":"1658756540","timeNano":"1658756540031720704.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T13:42:20.231+0000","rule":{"level":4,"description":"Docker: Network bridge disconnected","id":"87929","firedtimes":5,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658756540.600657","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"disconnect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658756540, \"timeNano\": 1658756540230828070}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"disconnect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"4bbeba6ee33a78b0278caab53c58005462eac4c5397387a910ebe42b79838226","name":"bridge","type":"bridge"}},"scope":"local","time":"1658756540","timeNano":"1658756540230828032.000000"}},"location":"Wazuh-Docker"}

</details>

**Remove the image**

<details><summary>Archives.json output</summary>

    {"timestamp":"2022-07-25T14:08:41.086+0000","rule":{"level":3,"description":"Docker: Image sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e untagged","id":"87919","firedtimes":1,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658758121.604141","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"untag\", \"id\": \"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e\", \"Type\": \"image\", \"Action\": \"untag\", \"Actor\": {\"ID\": \"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e\", \"Attributes\": {\"name\": \"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e\"}}, \"scope\": \"local\", \"time\": 1658758121, \"timeNano\": 1658758121086322467}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"untag","id":"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e","Type":"image","Action":"untag","Actor":{"ID":"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e","Attributes":{"name":"sha256:20fffa419e3aaca519dad480b21e86d9bcfbc7795abf9419adc5adce1198c95e"}},"scope":"local","time":"1658758121","timeNano":"1658758121086322432.000000"}},"location":"Wazuh-Docker"}

</details>

**Execute an application in a container**
Run `docker run -td ubuntu bash`

<details><summary>Archives.json output</summary>

    {"timestamp":"2022-07-25T14:21:24.949+0000","rule":{"level":3,"description":"Docker: Container elastic_ride created","id":"87901","firedtimes":1,"mail":false,"groups":["docker"],"pci_dss":["10.2.7"],"hipaa":["164.312.b"],"nist_800_53":["AU.14"],"tsc":["CC6.8","CC7.2","CC7.3"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658758884.607782","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"create\", \"id\": \"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"create\", \"Actor\": {\"ID\": \"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"elastic_ride\"}}, \"scope\": \"local\", \"time\": 1658758884, \"timeNano\": 1658758884948991620}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"create","id":"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f","from":"ubuntu","Type":"container","Action":"create","Actor":{"ID":"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f","Attributes":{"image":"ubuntu","name":"elastic_ride"}},"scope":"local","time":"1658758884","timeNano":"1658758884948991744.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T14:21:24.960+0000","rule":{"level":3,"description":"Docker: Network bridge connected","id":"87928","firedtimes":1,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658758884.607782","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"connect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658758884, \"timeNano\": 1658758884959754878}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"connect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f","name":"bridge","type":"bridge"}},"scope":"local","time":"1658758884","timeNano":"1658758884959754752.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T14:21:25.233+0000","rule":{"level":3,"description":"Docker: Container elastic_ride started","id":"87903","firedtimes":1,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658758885.609776","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"start\", \"id\": \"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"start\", \"Actor\": {\"ID\": \"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"elastic_ride\"}}, \"scope\": \"local\", \"time\": 1658758885, \"timeNano\": 1658758885233430907}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"start","id":"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f","from":"ubuntu","Type":"container","Action":"start","Actor":{"ID":"ba78279ad1972a8892af76bf4b37943a2dd7dfa4165abf216b2e13c8ce91bd4f","Attributes":{"image":"ubuntu","name":"elastic_ride"}},"scope":"local","time":"1658758885","timeNano":"1658758885233430784.000000"}},"location":"Wazuh-Docker"}

</details>

**Kill the container previously launched**
Run `docker kill $(docker ps -q | head -n1)`.

<details><summary>Archives.json output</summary>

    {"timestamp":"2022-07-25T14:27:04.989+0000","rule":{"level":7,"description":"Docker: Container quizzical_curran received the action: kill","id":"87924","firedtimes":1,"mail":false,"groups":["docker"],"gdpr":["IV_32.2"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658759224.1166009","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"kill\", \"id\": \"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"kill\", \"Actor\": {\"ID\": \"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99\", \"Attributes\": {\"image\": \"ubuntu\", \"name\": \"quizzical_curran\", \"signal\": \"9\"}}, \"scope\": \"local\", \"time\": 1658759224, \"timeNano\": 1658759224989177808}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"kill","id":"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99","from":"ubuntu","Type":"container","Action":"kill","Actor":{"ID":"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99","Attributes":{"image":"ubuntu","name":"quizzical_curran","signal":"9"}},"scope":"local","time":"1658759224","timeNano":"1658759224989177856.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T14:27:05.028+0000","rule":{"level":7,"description":"Docker: Container quizzical_curran received the action: die","id":"87924","firedtimes":2,"mail":false,"groups":["docker"],"gdpr":["IV_32.2"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658759225.1167087","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"status\": \"die\", \"id\": \"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99\", \"from\": \"ubuntu\", \"Type\": \"container\", \"Action\": \"die\", \"Actor\": {\"ID\": \"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99\", \"Attributes\": {\"exitCode\": \"137\", \"image\": \"ubuntu\", \"name\": \"quizzical_curran\"}}, \"scope\": \"local\", \"time\": 1658759225, \"timeNano\": 1658759225028136313}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"status":"die","id":"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99","from":"ubuntu","Type":"container","Action":"die","Actor":{"ID":"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99","Attributes":{"exitCode":"137","image":"ubuntu","name":"quizzical_curran"}},"scope":"local","time":"1658759225","timeNano":"1658759225028136192.000000"}},"location":"Wazuh-Docker"}

    {"timestamp":"2022-07-25T14:27:05.237+0000","rule":{"level":4,"description":"Docker: Network bridge disconnected","id":"87929","firedtimes":1,"mail":false,"groups":["docker"]},"agent":{"id":"000","name":"1ea6b53b44f8"},"manager":{"name":"1ea6b53b44f8"},"id":"1658759225.1168168","cluster":{"name":"wazuh","node":"wd_master_1"},"full_log":"{\"integration\": \"docker\", \"docker\": {\"Type\": \"network\", \"Action\": \"disconnect\", \"Actor\": {\"ID\": \"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc\", \"Attributes\": {\"container\": \"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99\", \"name\": \"bridge\", \"type\": \"bridge\"}}, \"scope\": \"local\", \"time\": 1658759225, \"timeNano\": 1658759225236440671}}","decoder":{"name":"json"},"data":{"integration":"docker","docker":{"Type":"network","Action":"disconnect","Actor":{"ID":"2b0214e31ead77766c1f35007ea575ab87ab42be7167faf47f05e6ddf29c4abc","Attributes":{"container":"49f807aaa693ae2ef47b1972905e91255a371135579c6a2f02ba910b249b1e99","name":"bridge","type":"bridge"}},"scope":"local","time":"1658759225","timeNano":"1658759225236440576.000000"}},"location":"Wazuh-Docker"}

</details>
